From d82230084e82679ad2a00ccf2053eef2f69d19ed Mon Sep 17 00:00:00 2001
From: Angela Stegmaier <angelabaker@ti.com>
Date: Wed, 30 Jun 2021 15:02:37 +0530
Subject: [PATCH 07/29] ti_config_fragments: multimedia.cfg: Add video decode

Create a new multimedia.cfg file and add d5520 vxd_dec v4l2
configurations to it.

Signed-off-by: Angela Stegmaier <angelabaker@ti.com>
Signed-off-by: Sidraya Jayagond <sidraya.bj@pathparntertech.com>
---
 ti_config_fragments/multimedia.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/ti_config_fragments/multimedia.cfg b/ti_config_fragments/multimedia.cfg
index f93841327787..f7e7ecf2a061 100644
--- a/ti_config_fragments/multimedia.cfg
+++ b/ti_config_fragments/multimedia.cfg
@@ -1,3 +1,7 @@
 ##################################################
 # TI Multimedia (Video Decode, Encode) Configs
 ##################################################
+# IMG D5500 v4l2 Driver Config Options
+
+CONFIG_MEDIA_SUPPORT=y
+CONFIG_VIDEO_IMG_VXD_DEC=m
-- 
2.17.1

